tasks/bump.js

```diff
@@ -96,14 +99,13 @@ module.exports = function(grunt) {
       });
     });
 
-
     // BUMP ALL FILES
     runIf(opts.bumpVersion, function() {
       grunt.file.expand(opts.files).forEach(function(file, idx) {
         var version = null;
-        var content = grunt.file.read(file).replace(VERSION_REGEXP, function(match, prefix, parsedVersion, suffix) {
-          gitVersion = gitVersion && parsedVersion + '-' + gitVersion;
```

Comment:
This part got left off so bump:git isn't working. https://github.com/vojtajina/grunt-bump/issues/129 Working on fixing it.